Description: spelling corrections in man pages
 Various spelling corrections after spellintian pass
 on man pages.
Author: Philippe Thierry
Last-Update: 2017-05-23
--- a/doc/manpages/wld.1
+++ b/doc/manpages/wld.1
@@ -1,6 +1,6 @@
 .TH WLD "1" "April 2017" "Witchcraft Compiler Collection" "User Commands"
 .SH NAME
-wld \- transform executables into shared libaries
+wld \- transform executables into shared libraries
 .SH SYNOPSIS
 .B wld
 [\fIoptions\fR] \fIfile\fR
